packages/jumpstarter-driver-android/jumpstarter_driver_android/driver/options.py

```
@@ -0,0 +1,202 @@
+from typing import Dict, List, Literal, Optional, Tuple
+
+from pydantic import BaseModel, Field
+
+
+class AdbOptions(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)